src/components/sky-nav/sky-nav.stories.mdx

```diff
@@ -1,47 +1,72 @@
 import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import menu from './demo/menu.json';
 import template from './sky-nav.twig';
-const menu = {
-  items: [
-    {
-      link: '/does',
-      title: 'What We Do',
-      current: true,
-    },
-    {
-      link: '/believes',
-      title: 'Our Approach',
-    },
-    {
-      link: '/made',
-      title: 'Our Work',
-    },
-    {
-      link: '/thinks',
-      title: 'Articles',
-    },
-    {
-      link: '/presents',
-      title: 'Speaking',
-    },
-    {
-      link: '/contributes',
-      title: 'Labs',
-    },
-    {
-      link: '/is',
-      title: 'Team',
-    },
-  ],
-};
-
-<Meta title="Components/Sky Nav" />
+
+<Meta
+  title="Components/Sky Nav"
+  parameters={{ paddings: { disabled: true } }}
+/>
 
 # Sky Nav
 
-This will eventually be our primary navigation component. Right now, we're just critiquing some markup updates in terms of their accessibility.
+**Note:** This component is currently a work in progress.
+
+Our primary navigation component, intended for the very top of a page layout. Its layout and behavior depends on the viewport width:
+
+- By default, menu items are hidden until toggled. They are revealed _above_ the brand and toggle, as if looking upward toward the sky.
+- In wider viewports, menu items are visible by default. They are displayed on multiple rows as needed.
+- At even wider viewports, the menu items are displayed in a single row.
+
+The Sky Nav is intended for use with our [our dark theme](/?path=/docs/themes--dark):
 
-We are tentatively using the same template properties as [Timber Menus](https://timber.github.io/docs/guides/menus/).
+<Preview>
+  <Story name="Dark" height="200px" parameters={{ themes: { disable: true } }}>
+    {template({ class: 't-dark', include_main_demo: true, menu })}
+  </Story>
+</Preview>
+
+But it works without a theme, too, just in case:
 
 <Preview>
-  <Story name="Markup (WIP)">{template({ menu })}</Story>
+  <Story name="Light" height="200px">
+    {template({ include_main_demo: true, menu })}
+  </Story>
 </Preview>
+
+## Elements
+
+- `c-sky-nav`: A containing `<div>` element.
+- `c-sky-nav__skip`: A link to the main content area, visibly hidden until focus. Works great with [our `c-button` pattern](/?path=/docs/components-button--elements).
```

Comment:
nit: Perhaps until "focused", might read better?

Reply:
I respectfully disagree, since it's more common when discussing interaction states to say "hover, active and focus" rather than "hovered, active, and focused." I also think this sounds better when read aloud (less sibilance on the end).